apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtaksvurderingRepository.kt

```diff
@@ -326,7 +327,21 @@ class VedtaksvurderingRepository(private val datasource: DataSource) : Transacti
                     ),
                 loggtekst = "Attesterer vedtak $behandlingId",
             )
-                .also { require(it == 1) }
+                .also {
+                    require(it == 1)
+                }
+
+            opprett(
+                query = """
+                    INSERT INTO outbox_vedtakshendelse (vedtakId, type) 
+                    SELECT v.id, 'ATTESTERT'
+                    FROM vedtak v
+                    WHERE v.behandlingId = :behandlingId
+                    """,
+                params = mapOf("behandlingId" to behandlingId),
+                loggtekst = "Lagt til innslag for attestert vedtak i outbox",
+            )
```

Comment:
ha i samme transaksjon slik at man ikke sender ut hendelse for noe som potensielt er rullet tilbake